### PR TITLE
Playwright reliability

### PR DESCRIPTION
**Pull request by heurtematte**
Add extra work on playwright script to make it reliable on different workstation: 
* Format python code with flake8
* Fix gh_signup script location
* Manage PASSWORD_STORE_DIR location for pass
* Migrate pass otp to oathtool
* Fix: token expiration date
* Add additional checks in all scripts
* Fix: secondary 2FA login page

### CODE REVIEW

File: github/playwright/gh_create_otterdog_token.py [hunk 1]
```diff
@@ -1,35 +1,52 @@
 import sys
 import common
 import pyperclip
-from playwright.sync_api import sync_playwright, expect
+from playwright.sync_api import sync_playwright, Error, expect
```
Comment (fredg02) [inline github/playwright/gh_create_otterdog_token.py:4]: Is `Error` used anywhere?

File: github/playwright/gh_signup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import requests
 import common
 import pyperclip
-from playwright.sync_api import sync_playwright, expect
+from playwright.sync_api import sync_playwright, Error, expect
```
Comment (fredg02) [inline github/playwright/gh_signup.py:7]: Is `Error` used anywhere?

**Review by fredg02 — APPROVED** [review]
As a general observation, doing code formatting and code changes in the same PR makes reading the complete diff more difficult. It's better, when viewing the changes commit by commit. GitHub diff not being the smartest, does not help. :unamused:

File: github/playwright/gh_create_otterdog_token.py (hunk 1 above)
Comment (heurtematte) [reply github/playwright/gh_create_otterdog_token.py:4]: Looks like not. I'll remove it.

**Comment by heurtematte** [comment]
The code formatting should have been done first, right? :slightly_smiling_face: 

I made atomic commits, making it easier to focus on specific modifications and I think GitHub's Diff tool handles it well.
Also the size of this PR is quite reasonable to review.

Maybe reviewing a PR too late isn't good for the eyes. :smile: 

Anyway, if it's really an issue, I'm happy to create another PR.

**Comment by fredg02** [comment]
I was mostly complaining about the GitHub diff. Even in the morning it looks like crap. No need for another PR.

**Pull request merged**